README.md

```diff
@@ -36,6 +36,22 @@ To check the minimum version of Firefox needed for a given example,
 see the `strict_min_version` part of the [applications key](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications)
 in the example's manifest.json file.
 
+## Installation
+
+There are a couple ways to try out the example extensions in this repository.
+
+1. Open Firefox and load `about:debugging` in the URL bar. Click the
+   [Load Temporary Add-on](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Temporary_Installation_in_Firefox)
```

Comment:
Actually I wondered if we could embed it in the README itself, but a link is fine.